README.md

```diff
@@ -308,6 +308,105 @@ Descope SDK is automatically refreshes the session token when it is about to exp
 If the Descope project settings are configured to manage tokens in cookies.
 you must also configure a custom domain, and set it as the `baseUrl` prop in the `AuthProvider` component. See the above [`AuthProvider` usage](https://github.com/descope/react-sdk#wrap-your-app-with-auth-provider) for usage example.
 
+### Widgets
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, he needs to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `User Management` widget will let you embed a user table in your site to view and change information, exposing the following fields:
+
+`LoginId`, `Status`, `Name`, `Email`, `Phone`, `Roles`
+
+The widget also lets you:
+
+- Create a new user.
+- Edit an existing user.
+- Activate / disable an existing user.
+- Reset an existing user's password.
+- Reset an existing user's passkey.
+- Delete an existing user.
+
+###### Implementation
+
+```js
+import { UserManagement } from '@descope/react-sdk';
+...
+			<UserManagement
+                widgetId="user-management-widget"
```

Comment:
there seems to have a lot of whitespace here

same to the bellow

Reply:
@ruvenzx it still doesn't look good
![image](https://github.com/descope/react-sdk/assets/10514677/d4ffc5a1-a3b7-4847-a857-88405e4dd94c)